src/pspm_check_model.m

```diff
@@ -399,11 +384,145 @@
     elseif ~iscell(model.method)
       warning('Method needs to be a char or cell array'); return;
     end
-
 end
 
+%% 9. TAM-specific checks
+% -------------------------------------------------------------------------
+if strcmpi(modeltype, 'tam')
+ if (~isfield(model, 'window') || isempty(model.window) || ~isnumeric(model.window))
+      warning('ID:invalid_input', 'Window is expected to be a numeric value.'); return;
+ elseif
```

Comment:
I am not sure if it means `else` here, or there would like to be a condition following `elseif`?